include/HubFramework/HUBViewController.h

```diff
@@ -128,6 +128,15 @@ NS_ASSUME_NONNULL_BEGIN
   didDisappearFromView:(UIView *)componentView;
 
 /**
+ *  Sent to a Hub Framework view controller's delegate when a component view will be reused
+ *
+ *  @param viewController The view controller in which a component view will be reused
+ *  @param componentView The component view that will be reused
+ */
+- (void)viewController:(HUBViewController *)viewController
+willReuseComponentWithView:(UIView *)componentView;
```

Comment:
🎨 Super optional very nitpicky style comment: Align this with the line above like this:
```objective-c
- (void)viewController:(HUBViewController *)viewController
        willReuseComponentWithView:(UIView *)componentView;
```